stages/10-openebs-upgrade-check/upgrade-replica-pod-pending-check.md

```diff
@@ -0,0 +1,132 @@
+# Upgrade the application with 3 replicas when one replica pod is not running
```

Comment:
@amitbhatt818 lets redo the readme in a different way. Let us explain the assumptions & expectations clearly.